index-handlers/src/main/java/no/sikt/nva/nvi/index/aws/CandidateQuery.java

```diff
@@ -79,8 +80,8 @@ public CandidateQuery(CandidateQueryParameters params) {
 
     public static Query matchAtLeastOne(Query... queries) {
         return new Query.Builder()
-            .bool(new BoolQuery.Builder().should(Arrays.stream(queries).toList()).build())
-            .build();
+                   .bool(new BoolQuery.Builder().should(Arrays.stream(queries).toList()).build())
+                   .build();
```

Comment:
Note that if you are looking or OR queries, this is way to do it in Opensearch...

```
QueryBuilders.disMaxQuery()
            .add(new TermQueryBuilder(field1, value))
           .add(new TermsQueryBuilder(field2, values))
           .....

```

Reply:
hit on one or more of these Terms will include the document in the result

Reply:
should isn't an OR query, it just boost queries that hits.

Reply:
Thanks for the tip. I haven't changed this code and the tests are green, so merging this PR. Can take a closer look at it and make a new PR with modifications.

Reply:
This is only a problem when you have several keys to filter on, and you need something like this
`key=X & ( key2=Y || key2=Z ) & key3=BZ`

`key=X & ( key2=Y || key2=Z ) & key3=BZ || key4=CX`

what does the last OR mean?